cyvlfeat/sift/phow.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import division
+import math
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import matplotlib.colors as colors
+import dsift
+
+
+def phow(image, verbose=False, fast=True, sizes=[4, 6, 8, 10], step=2, color='gray',
+         floatdescriptors=False, magnif=6, window_size=1.5, contrastthreshold=0.005):
+    # -------------------------------------------------------------------
+    #  Parse the arguments
+    # -------------------------------------------------------------------
+    dsiftOpts = ['norm', 'window_size', window_size]
```

Comment:
Throughout this file, please change CamelCase to underscore -> `dsiftOpts` to `dsift_opts`.